PSReadLine/PublicAPI.cs

```diff
@@ -235,5 +235,34 @@ public static bool TryGetArgAsInt(object arg, out int numericArg, int defaultNum
             numericArg = 0;
             return false;
         }
+
+        /// <summary>
+        /// Accept the suggestion text if there is one.
+        /// </summary>
+        public static void AcceptSuggestion()
+        {
```

Comment:
Is this meant to be bindable? If so, it has the wrong signature.

Reply:
I think it's desirable to make it bindable. I will update the code.